hbase-server/src/test/java/org/apache/hadoop/hbase/replication/regionserver/TestWALEntryStream.java

```diff
@@ -378,37 +380,35 @@ public void testReplicationSourceWALReaderThread() throws Exception {
   }
 
   @Test
-  public void testReplicationSourceUpdatesLogPositionOnFilteredEntries() throws Exception {
+  public void testReplicationSourceWALReaderThreadRecoveredQueue() throws Exception {
```

Comment:
Why this test has been changed from it's original purpose of checking for upating wal position when no edits targeted to replication are read? Also the name does not seem accurate, it does not seem to create a recovered queue scenario.

Reply:
I removed `testReplicationSourceUpdatesLogPositionOnFilteredEntries`, because the behavior of the reader is changed. (no updates every filtered entry).

And, i made recovered queue scenario by setting queue info as recovered queue. `getQueueInfo("1-1")`
https://github.com/apache/hbase/pull/749/files/114aa1b1a7b9919c5429fadcb74079cd08629513#diff-05e8e2a626166f52e5737f8bcdc49e39R401

If it is not well recognized as intended, let me add comments or `getRecoveredQueueInfo()`?

Reply:
I changed the method name to `getRecoveredQueueInfo()`